### DISCUSSION TITLE
How do you use the Figma plugin?

### DISCUSSION THREAD

**Discussion by treeder** [material-foundation/material-theme-builder#211] [Q&A] [answered]
I can't figure it out for the life of me (I'm not very familiar with Figma). 

The docs say there should be some Create Theme button, but I don't see anything. 

![image](https://github.com/material-foundation/material-theme-builder/assets/75826/3014b6a9-3509-4619-90e1-2ae7a60d0742)

Any guidance would be great.

**Comment by margeeta** [comment, marked as answer]
Hello! 
The screenshot you posted is of the playground file, which has abbreviated instructions to try out the plugin. 

The plugin can be opened from the plugin panel. (Icon in the top bar with little shapes, then select **Plugins**.) 
Then click **RUN**
![Screenshot 2023-09-11 at 3 13 05 PM](https://github.com/material-foundation/material-theme-builder/assets/10263978/e9691de9-3a3f-4316-b5b3-159be9a35633)
After, it will appear as recent plugin: 
![Screenshot 2023-09-11 at 3 08 28 PM](https://github.com/material-foundation/material-theme-builder/assets/10263978/82779a77-29ec-4c10-9d4c-350612234327)
